### DISCUSSION TITLE
Obelisk Share Puzzle between co-op players

### DISCUSSION THREAD

**Discussion by denierlexiese** [ihhub/fheroes2#5691] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Can we have the ability to share Obelisk puzzle pieces between coop players? ( Removal on the dig map)  On bigger maps it's hard to traverse the whole map. especially in coop multiplayer mode so the likellyhood of finding the artificact is slim.  It would act an extra piece of strategy in coop multipayer games.

Thank you.

### Describe the possible solution

Shared puzzle pieces between coop teammates.

### Additional info

Thank you for your time and effort.

**Comment by ihhub** [comment]
I am converting this into a conversation topic as many such ideas should go through discussions.
